install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+# This is a convenience script that can be downloaded from GitHub and
+# piped into "sh" for conveniently downloading the latest GitHub release
+# of the pizza CLI:
+#
+# curl -fsSL https://raw.githubusercontent.com/open-sauced/pizza-cli/main/install.sh | sh
+#
+# Warning: It may not be advisable to pipe scripts from GitHub directly into
+# a command line interpreter! If you do not fully trust the source, first
+# download the script, inspect it manually to ensure its integrity, and then
+# run it:
+#
+# curl -fsSL https://raw.githubusercontent.com/open-sauced/pizza-cli/main/install.sh > install.sh
+# vim install.sh
+# ./install.sh
+
+PIZZA_REPO="open-sauced/pizza-cli"
+ARCH=""
+
+# Detect architecture
+case "$(uname -m)" in
+    x86_64) ARCH="x86_64" ;;
+    arm64)  ARCH="arm64"  ;;
+    *)      echo "Unsupported architecture"; exit 1 ;;
+esac
+
+# Detect OS system type. Windows not supported.
+case "$(uname -m)" in
```

Comment:
```suggestion
# Detect OS system type. Windows not supported.
case "$(uname -s)" in
```

I think this is a typo. Since `-m` is used to get the architecture, which is done above. The man page of `uname` suggests `-s` to get the OS name.

Reply:
Whoops! Good catch. Yes, this should be the OS's type, not arch 👍🏼